requirements.txt

```diff
@@ -4,3 +4,4 @@ scikit-learn
 numpy
 gensim>=0.12.3  # LDA's show topics unified in 0.12.3
 setuptools-git
+newspaper
```

Comment:
Isn't `newspaper` a [Python 2 library](https://github.com/codelucas/newspaper/tree/python-2-head)? Shouldn't we be using a [Python 3 version](https://github.com/codelucas/newspaper/tree/master) – from a `newspaper3k` packet?